syncstorage/storage/sql/queries_generic.py

```diff
@@ -92,6 +93,46 @@
 DELETE_ITEMS = "DELETE FROM %(bso)s WHERE userid=:userid "\
                "AND collection=:collectionid AND id IN %(ids)s"
 
+CREATE_BATCH = "INSERT INTO batch_uploads (batch, userid, collection) "\
+                     "VALUES (:batch, :userid, :collection)"
+
+VALID_BATCH = "SELECT batch FROM batch_uploads WHERE batch = :batch " \
+                    "AND userid = :userid AND collection = :collection"
+
+APPEND_ITEMS_TO_BATCH = "INSERT INTO %(bui)s "\
```

Comment:
This query appears to be unused, I assume because of use of the `insert_or_update` helper.